content/en/cloud_cost_management/budgets.md

```diff
@@ -41,27 +75,33 @@ You cannot view budget versus actuals before 15 months, since cost metrics are r
 
 - You can view the budget status for the **current month**:
 
-   {{< img src="cloud_cost/budgets/budget-status-month.png" alt="Budget Status View: view current month">}}
+   {{< img src="cloud_cost/budgets/budget-status-month-1.png" alt="Budget Status View: view current month">}}
```

Comment:
I recommend redacting the names from this screenshot, as well as from the ones that follow.